packages/app/components/Queue/TAQueueList.tsx

```diff
@@ -246,7 +246,9 @@ interface QueueProps {
 function QueueQuestions({ questions, isHelping, onOpenCard }: QueueProps) {
   return (
     <div data-cy="queueQuestions">
-      {questions.length === 0 ? (
+      {questions.filter(
+        (question) => question.status !== OpenQuestionStatus.TADeleted
+      ).length === 0 ? (
```

Comment:
Could store this filtered questions array into a constant and just use this both places

Reply:
gonna handle this in my ticket w/ Olivia